docs/functions/functions.json

```diff
@@ -1346,6 +1470,30 @@
 				true
 			]
 		},
+		{
+			"name": "contains",
+			"id": "contains",
+			"unary": false,
+			"category": "math",
+			"role": "function",
+			"description": "Check if a collection, group, list, or text contains a needle. For example, `contains(group(name:\"Kenneth\", age:17, food:\"rice\"), \"rice\")` returns `true`.\nIf both haystack and needle are a list, `contains()` checks if the first item of the haystack contains the first item of the needle, and so on, and returns a list of corresponding results. For example, `contains(list(\"Kenneth\", \"Verou\"), list(\"Ken\", \"Lea\"))` will return `list(true, false)`.",
```

Comment:
A needle? What is a needle? Is this about making web apps or sewing? :) 
"Check if a value contains some given text" or something like that may make more sense.

Also, you may want a simpler example as the first one, with just text.

Reply:
Something like: `contains("Kenneth", "Ken")` returns `true`? If so, I feel like it won't demonstrate the core functionality of `contains` very well. Some viewers might think, "Wait, isn't that similar to `search`"?